llm_perf/update_llm_perf_leaderboard.py

```diff
@@ -31,12 +34,19 @@ def gather_benchmarks(subset: str, machine: str):
 
 
 def update_perf_dfs():
-    for subset in ["unquantized", "bnb", "awq", "gptq"]:
-        for machine in ["1xA10", "1xA100", "1xT4"]:
-            try:
-                gather_benchmarks(subset, machine)
-            except Exception:
-                print(f"Subset {subset} for machine {machine} not found")
+    """
+    Update the performance dataframes for all machines
+    """
+    for machine in ["1xA10", "1xA100", "1xT4", "32vCPU-C7i"]:
+        for backend in ["pytorch"]:
+            for hardware in ["cuda", "cpu"]:
+                for subset in ["unquantized", "bnb", "awq", "gptq"]:
```

Comment:
To prevent such a cumbersome logic, I think this could be a solution https://github.com/huggingface/optimum-benchmark/pull/258